src/output/outputMod.F90

```diff
@@ -1782,17 +1832,119 @@ subroutine storeSurfsolInBuffer(sps, buffer, nn, blockID,   &
        !================================================================
 
     case (cgnsCp)
-
+       ! mham:
+       ! [two] l 68, constants.F90
+       ! [gammaInf] l. 88 initializeFlow.F90
+       ! nonDimensional constant?
+       !   real(kind=realType), parameter :: two   = 2.0_realType
+       ! [MachCoef] l. 520 inputParam.F90
+       ! Mach number used to compute coefficients;
+       ! only relevant for translating geometries.
+       !
        ! Factor multiplying p-pInf
-
-       fact = two/(gammaInf*pInf*MachCoef*MachCoef)
+       fact = two/(gammaInf*pInf*MachCoef*MachCoef) ! = uInf2*rhoInf 
+       ! Above line makes mores sense in light of l.116 in initializeFlow.F90.
+       ! clearly fact = uInf2*rhoInf 
+       !
+       ! mham:
+       ! from Sandy; the two/(gammaInf*pInf*MachCoef*MachCoef) should be equiva-
+       ! lent to 0.5*rho*v^2... But I need the rotational speed to get the
+       ! apparent velocity of the blade.
+       !
+       ! mham: check if fact == factDIM / rhoRef / uRef / uRef
+       ! factDim = two / (rhoInfDim*((uInf*uRef)**2))
+       ! below the check is clearly ok. I simply take the dimensions out
+       ! of my factDIM and get the ADflow fact...So far so good. Now add (r*w)^2
+       !PRINT*,'OK'
+       !PRINT*,'fact',fact
+       !PRINT*,'factDIM w.o. dims',(factDIM * rhoRef * uRef * uRef) ! U squared
+       !
+       ! (r*w)**2
 
        do j=rangeFace(2,1), rangeFace(2,2)
           do i=rangeFace(1,1), rangeFace(1,2)
              nn = nn + 1
-             buffer(nn) = fact*(half*(pp1(i,j) + pp2(i,j)) - pInf)
-          enddo
+             !buffer(nn) = fact*(half*(pp1(i,j) + pp2(i,j)) - pInf)
+             ! mham Cp normalisation:
+             rrate_=cgnsdoms(1)%rotrate
+             r_(1) =   (half*(xx1(i,j,1) + xx2(i,j,1)))
+             r_(2) =   (half*(xx1(i,j,2) + xx2(i,j,2)))
+             r_(3) =   (half*(xx1(i,j,3) + xx2(i,j,3)))
+             ! calc cross-product between rotation rate and r_
+             wCrossR(1) = rrate_(2)*r_(3) - rrate_(3)*r_(2)
+             wCrossR(2) = rrate_(3)*r_(1) - rrate_(1)*r_(3)
+             wCrossR(3) = rrate_(1)*r_(2) - rrate_(2)*r_(1)
+             rot_speed = SQRT(wCrossR(1)**2 +wCrossR(2)**2 +wCrossR(3)**2 )
+             buffer(nn) = ((half*(pp1(i,j) + pp2(i,j)) - pInf)*pRef) &
+                  / (half*(rhoInfDim)*((uInf*uRef)**2 + (rot_speed)**2 ))
+             ! Note, that we take rrate_(1) since we rotate 
```

Comment:
these comments should be useful, they are just explanations of what is happening. they are not commented out code, they are just text explanations. maybe add a note saying that these are not commented out computations, but rather explanations of what is happening above

Reply:
added consistent comment